cypress/integration/issue-list.spec.ts

```diff
@@ -95,5 +100,65 @@ describe("Issue List", () => {
           });
         });
     });
+
+    it("renders issues with resolved status only", () => {
+      cy.dataCy("filter-by-status").click();
+      cy.wait(2000);
```

Comment:
Did you try the tests without waiting manually? Cypress should wait by itself so maybe these aren't necessary.